src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -335,5 +335,94 @@ public static Navigator GetDefaultNavigator(Window window)
 
             return defaultNavigator;
         }
+
+        /// <summary>
+        /// Shows a dialog by pushing a page containing dialog to default navigator.
+        /// </summary>
+        /// <param name="content">The content of Dialog.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static void ShowDialog(View content = null)
+        {
+            var window = NUIApplication.GetDefaultWindow();
+            var defaultNavigator = window.GetDefaultNavigator();
```

Comment:
I have question that is not directly related with this patch.
Does the method name `GetDefaultNavigator` means there are multiple Navigator in a window?

Reply:
Yes, it is possible if a user creates multiple Navigator instances in a window.
e.g.
```
var window = NUIApplication.GetDefaultWindow();

var navigator = new Navigator();
window.Add(navigator);

var navigator2 = new Navigator();
window.Add(navigator2);
```

If window.GetDefaultNavigator() is called for the first time, then Navigator instance is created and stored in Dictionary with the window.
If window.GetDefaultNavigator() is called again, then the stored Navigator instance is returned.